src/client.rs

```diff
@@ -48,6 +48,14 @@ pub struct ClientFlow {
     receive_response_state: ReceiveState<ResponseCodec>,
 }
 
+impl Debug for ClientFlow {
+    fn fmt(&self, f: &mut Formatter) -> std::fmt::Result {
+        f.debug_struct("ClientFlow")
+            .field("handle_generator", &self.handle_generator)
+            .finish_non_exhaustive()
+    }
+}
+
```

Comment:
I don't think it's necessary, but you can keep it. But why only for `ClientFlow` and not `ServerFlow`?